docs/guide/getting-started.md

```
@@ -190,6 +286,150 @@ const operationId = route.data.params.operationId
 
 :::
 
+</details>
+
+<details>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: details

(MD033, no-inline-html)